### ISSUE TITLE
[BUG] [Linter] T002 false positive

### ISSUE THREAD

**Issue by heitorado** [Riverside-Healthcare/djLint#349]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: e.g. ubuntu 22.04
 - Python Version (``3.10.4``)
 - djLint Version (``1.11.1``)
 - template language: `django`

## Issue

Linter is reporting T002 rule violation when using variables on `extends`. I believe that it should only report when the parameter is wrapped in single quotes, since variables can also be used.

## How To Reproduce

Add `base_template` to your django view context and assign it a string with the template name.

Then, on the template:
```
{% extends base_template %}
{% block title %}
    My Page Title
{% endblock title %}
{% block content %}
  <div> Something here... </div>
{% endblock content %}

```

Notice that T002 is reported for the first line, but in this case it does not make sense to wrap it in double quotes.
